docs/sphinx/sections/api/apidocs/libraries/dagster-aws.rst

```diff
@@ -38,6 +32,17 @@ S3
 .. autoconfigurable:: dagster_aws.s3.s3_pickle_io_manager
   :annotation: IOManagerDefinition
 
+File Manager (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
We normally use dashes, not carets

Reply:
i believe carets are for subheadings, but I could be wrong

Reply:
Ah oops ok I believe you